microsoft-edge/toc.yml

```diff
@@ -816,77 +815,75 @@
             - name: Native messaging
               href: extensions-chromium/developer-guide/native-messaging.md
 
+            - name: Defining match patterns for an extension to access file URLs
+              href: extensions-chromium/developer-guide/match-patterns.md  # moved from /enterprise/
+
         - name: Manifest
           items:
             - name: Overview and timelines for migrating to Manifest V3
               href: extensions-chromium/developer-guide/manifest-v3.md
-              displayName: 
 
             - name: Migrate an extension from Manifest V2 to V3
               href: extensions-chromium/developer-guide/migrate-your-extension-from-manifest-v2-to-v3.md
-              displayName: 
 
             - name: Manifest file format for extensions
               href: extensions-chromium/getting-started/manifest-format.md
-              displayName: 
 
         - name: Publish to the Add-ons website
           items:
-            - name: Register as an extension developer
+            - name: Register as a Microsoft Edge extension developer
               href: extensions-chromium/publish/create-dev-account.md
 
-            - name: Publish extensions using a GitHub account
+            - name: Publish a Microsoft Edge extension by using a GitHub account
               href: extensions-chromium/publish/github.md
 
             - name: Add users to the Microsoft Edge program
               href: extensions-chromium/publish/aad-account.md
 
-            - name: Publish an extension
+            - name: Publish a Microsoft Edge extension
               href: extensions-chromium/publish/publish-extension.md
 
-            - name: Update an extension
+            - name: Publish and update extensions in the Microsoft Edge Add-ons website
+              href: extensions-chromium/publish/hosting-and-updating.md  # moved from /enterprise/
+
+            - name: Update a Microsoft Edge extension
               href: extensions-chromium/publish/update-extension.md
 
-            - name: Submission states
+            - name: Automatically update extensions in Microsoft Edge
+              href: extensions-chromium/publish/auto-update.md  # moved from /enterprise/
```

Comment:
@sonakshimore See Note at top of the upstream doc, https://developer.chrome.com/docs/apps/autoupdate/

Reply:
Have asked an engineering team mate to review this document. https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/auto-update?branch=pr-en-us-1985

But I recommend we go ahead with merging this - and sharing back the published document link as soon as its merged. We can take his inputs into consideration later.